suggestion2/sugCreate.cc.lua

```diff
@@ -0,0 +1,88 @@
+{{/*
+Made by: Crenshaw#1312
+
+Trigger Type: Regex
+Trigger: .*
+		
+Note: read the other file for more info :)
+~~~
+Note: ONLY ALLOW THIS TO RUN IN ONE CHANNEL (this should be your suggestion chanel)
+Note: Allow people to type in your suggestion channel
+~~~
+Note: You can change your upvote/downvote in line 75 (on Github)
+*/}}
+
```

Comment:
Cooldown should also be an option? I do not see any such option in code. Usually many would want to prevent spamming of suggestions.

Reply:
Cooldown can be done by the slowmode of the channel itself